process/coordinator/process.go

```diff
@@ -748,6 +749,29 @@ func (tc *transactionCoordinator) CreateMbsAndProcessCrossShardTransactionsDstMe
 	return createMBDestMeExecutionInfo.miniBlocks, createMBDestMeExecutionInfo.numTxAdded, allMBsProcessed, nil
 }
 
+func (tc *transactionCoordinator) requestMissingMiniBlocks(mbsInfo []*data.MiniBlockInfo) {
```

Comment:
functionality of _requestMissingMiniBlocks_  and _RequestMiniBlocks_ is very similar, could the refactored with a method that gets a list of miniblockHashes and does the request for those and be called from both other methods.

Reply:
Right, will do this refactor